draft-ietf-quic-http.md

```diff
@@ -881,12 +885,12 @@ the DUPLICATE_PUSH.
 
 ### Reserved Frame Types {#frame-grease}
 
-Frame types of the format `0xb + (0x1f * N)` are reserved to exercise the
-requirement that unknown types be ignored ({{extensions}}). These frames have no
-semantic value, and can be sent when application-layer padding is desired. They
-MAY also be sent on connections where no request data is currently being
-transferred. Endpoints MUST NOT consider these frames to have any meaning upon
-receipt.
+Frame types of the format `0x1f * N` for non-zero values of N are reserved to
+exercise the requirement that unknown types be ignored ({{extensions}}). These
+frames have no semantic value, and can be sent when application-layer padding is
+desired. They MAY also be sent on connections where no request data is currently
```

Comment:
Not a new problem but I wonder if request data can be simplified to data, which covers all cases such as QPACK instructions, response data, control data etc.